src/main/kotlin/no/nav/meldeplikt/meldekortservice/service/ArenaOrdsService.kt

```diff
@@ -66,16 +66,16 @@ class ArenaOrdsService(
     }
 
     suspend fun kopierMeldekort(meldekortId: Long): Long {
```

Comment:
`
suspend fun kopierMeldekort(meldekortId: Long): Long {
        try {
            val responseMedNyMeldekortId = ordsClient.post<String>("${env.ordsUrl}$ARENA_ORDS_KOPIER_MELDEKORT") {
                setupOrdsRequest(meldekortId)
            }

            val nyMeldekortId = mapFraXml(responseMedNyMeldekortId, KopierMeldekortResponse::class.java).meldekortId
            defaultLog.info("Meldekort med id $nyMeldekortId er opprettet for korrigering. Kopiert fra meldekort med id $meldekortId")
            return nyMeldekortId

        } catch (e: Exception) {
            defaultLog.warn("Feil ved opprettelse av meldekort for korrigering! Meldekort med id $meldekortId har ikke blitt kopiert.", e)
        }

        return 0
    }
`